pl_bolts/models/self_supervised/swav/transforms.py

```diff
@@ -102,13 +110,21 @@ class SwAVEvalDataTransform(SwAVTrainDataTransform):
     def __init__(
         self,
         normalize=None,
-        size_crops: List[int] = [96, 36],
-        nmb_crops: List[int] = [2, 4],
-        min_scale_crops: List[float] = [0.33, 0.10],
-        max_scale_crops: List[float] = [1, 0.33],
+        size_crops: List[int] = None,
+        nmb_crops: List[int] = None,
+        min_scale_crops: List[float] = None,
+        max_scale_crops: List[float] = None,
         gaussian_blur: bool = True,
         jitter_strength: float = 1.0,
     ):
+        if size_crops is None:
+            size_crops = [96, 36]
+        if nmb_crops is None:
+            nmb_crops = [2, 4]
+        if min_scale_crops is None:
+            min_scale_crops = [0.33, 0.10]
+        if max_scale_crops is None:
+            max_scale_crops = [1, 0.33]
```

Comment:
you can make them tuples to avoid this boilerplate

Reply:
yes, that is what I did in the commit just before your comment :] 93cb51899b9fa5818c773dbc21a0d8c9e02e8f07